modules/framework/galasa-parent/galasa-boot/src/main/java/dev/galasa/boot/Launcher.java

```diff
@@ -653,4 +655,27 @@ public void setStoresFromEnvironmentVariables(Environment env, Properties bootst
             bootstrap.setProperty("framework.auth.store",authStore);
         }
     }
+
+    private MonitorConfiguration getResourceMonitorConfigFromEnvironment(Environment env) {
```

Comment:
I might be tempted to move this into the MonitorConfiguration itself, expecially as the env is already passed to it... to keep parsing as close to the code using it, and unit tests of the parsing together...

Reply:
Moved in https://github.com/galasa-dev/galasa/pull/236/commits/fa36d081355b9cd4e5ef366e28cde2f8089f0b53